io.github.btpf.alexandria.yml

```diff
@@ -0,0 +1,56 @@
+---
+id: io.github.btpf.alexandria
+command: "alexandria"
+rename-icon: "alexandria"
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+# Use rust-nightly to get once_cell working on older rust runtime compatible with org.gnome.platform 42
```

Comment:
so you need a compiler more recent than the platform you are limited to. 

My recommendation is to use a supported runtime/platform and compile webkit gkt until Tauri fix their side.

Reply:
I am not a fan of compiling webkit gtk because it will make testing and iterating on the flatpak a very tedious process for my machine. I understand that the runtime is unsupported, but this situation should only be temporary until I am ready to move to Tauri V2.